### PR TITLE
docs(web-modeler): document helm values for custom OIDC

### PR DESCRIPTION
**Pull request by wollefitz**
## Description

Relates to https://github.com/camunda/web-modeler/issues/7077

## When should this change go live?

- [x] This change is not yet live and should not be merged until the `8.4` minor release (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] ~I have added changes to the relevant `/versioned_docs` directory,~ or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), ~or they are not for **future versions**.~
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md [hunk 1]
```diff
@@ -86,6 +86,10 @@ global:
         clientId: <Client ID from step 2>
         audience: <Audience from step 2>
         existingSecret: <Client secret from step 2>
+      webModeler:
+        clientId: <Client ID from step 2>
```
Comment (Ben-Sheppard) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:90]: Should there also be a `clientSercret` variable here?

Comment (wollefitz) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:90]: No, Web Modeler doesn't require a client secret.

Comment (Ben-Sheppard) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md:90]: 🤦🏻  of course

**Review by Ben-Sheppard — APPROVED** [review]
Thank you for adding this, the content makes sense to me. The only thing that I'm not sure on is if we should link to somewhere regarding the `publicApiAudience` selection? But thats somethign for @christinaausley maybe

**Comment by wollefitz** [comment]
> But thats somethign for @christinaausley maybe

Thanks! I added @christinaausley as reviewer

**Comment by christinaausley** [comment]
@Ben-Sheppard Where would you propose linking? Are we missing documentation or content for our existing documentation that would be useful here if it is not intuitive for the users?

**Comment by Ben-Sheppard** [comment]
> @Ben-Sheppard Where would you propose linking? Are we missing documentation or content for our existing documentation that would be useful here if it is not intuitive for the users?

Is "I don't know" a valid answer here? :D - I suppose as a user reading these docs I'm not sure _why_ that valid needs to be different or what even the publicApiAudience is, though maybe if I am an end user I already know?

**Comment by akeller** [comment]
I'm ok with merging as-is. If you read the guide from top to bottom, the user should be informed enough. That said, we can always improve in the future because I don't see an obvious place to link for more info. The guide should be consumed fully.

**Comment by wollefitz** [comment]
> I'm ok with merging as-is

@christinaausley @akeller Thanks for your input! Feel free to merge this PR then whenever it fits.

**Pull request merged**